app/models/katello/glue/candlepin/product_content.rb

```diff
@@ -1,15 +0,0 @@
-module Katello
```

Comment:
If we're removing this what's the plan for https://github.com/Katello/katello/blob/13b438b13726cfdb81e9f130fc289d8b09e5905d/app/models/katello/candlepin/content.rb#L56

Reply:
ahhh, right!  let me see what i can do